src/stringnumber.ts

```diff
@@ -9,8 +9,9 @@ import { Modifier } from './modifier';
 import { ModifierContextState } from './modifiercontext';
 import { Note } from './note';
 import { Renderer } from './renderer';
+import { isStaveNote } from './stavenote';
 import { Stem } from './stem';
-import { isStaveNote, isStemmableNote } from './typeguard';
+import { isStemmableNote } from './stemmablenote';
```

Comment:
Removed some circular dependencies by moving the `isElementXXX()` typeguard methods into their respective classes. So `isStaveNote()` is now defined in stavenote.ts.